content/methods/decide/service-blueprint.md

```diff
@@ -34,7 +34,8 @@ method:
 
 ## Example from 18F{#ex-service}
```

Comment:
**comment (non-blocking)**: Should this heading be plural now since there are two examples?

Reply:
@neilmb good idea! is there a way to update that without making a new pull request? (I'm new to this!)

Reply:
@neilmb I think I updated the title correctly within this same pull request? 🤞